pkg/cri/server/container_create_linux.go

```diff
@@ -260,6 +261,15 @@ func (c *criService) containerSpec(
 
 	supplementalGroups := securityContext.GetSupplementalGroups()
 
+	// Get RDT class
+	rdtClass, err := rdt.ContainerClassFromAnnotations(config.GetMetadata().GetName(), config.Annotations, sandboxConfig.Annotations)
+	if err != nil {
+		return nil, errors.Wrap(err, "invalid RDT class")
```

Comment:
two types of errors here.. invalid class and not enabled.. needs a generic string on the wrap

Reply:
Changed wording to `failed to set RDT class` which covers all types of errors